UI/index.html

```diff
@@ -1,37 +1,41 @@
-
 <!Doctype html>
 <html>
+
 <head>
 	<meta charset="UTF-8">
 	<title>{{{config.userName}}}'s Dairy</title>
 	<link href='http://fonts.googleapis.com/css?family=Adamina' rel='stylesheet' type='text/css'>
 	<link href='http://fonts.googleapis.com/css?family=Merriweather:400,300' rel='stylesheet' type='text/css'>
 	<link href='http://fonts.googleapis.com/css?family=Josefin+Sans:300,400' rel='stylesheet' type='text/css'>
 	<link href='http://fonts.googleapis.com/css?family=Raleway:300,400,700' rel='stylesheet' type='text/css'>
+	<link href='http://fonts.googleapis.com/css?family=Berkshire+Swash' rel='stylesheet' type='text/css'>
 	<link rel="stylesheet" href="style.css">
 </head>
-<body>
-	<header>
-		{{{config.userName}}}'s Dairy.
-		<div id="info" contenteditable="true"></div>
-    </header>
-    <h1 contenteditable="true"> 
-        <!-- Removing dropcap for now messing with css.. Can anyone fix this-->
-        <!-- <span id="dropcap">T</span>  -->
-        Title</h1>   
-    <br>
-	<main contenteditable="true">
-		<p>
-                  Hello world.
-		</p>
 
-	</main>
-    <script src="../dropcap.min.js"></script>
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
-    <script src="/dairy.js"></script>
-	<!-- <script>
-		var dropcap = document.getElementById("dropcap");
-		window.Dropcap.layout(dropcap, 2);
-	</script> -->
+<body>
+	<section id="headerBar">
+		<header>
+			{{{config.userName}}}'s Dairy.
+			<div id="info" contenteditable="true"></div>
+		</header>
+	</section>
+	<hr>
+	<section id="Title">
+		<div contenteditable="true">
+			<h1>
+				<span class="dropcaps">T</span>his is a test.</h1>
+	</section>
+	<br>
+	<section id="dairyContent">
+		<div contenteditable="true">
+			<p>
+				<span class="dropcaps">H</span>ello world</p>
```

Comment:
@mdalmazan  If I delete the place holder text ("Hello World") and start typing all the text comes under dropcaps class. This would be painful experience for the user. I think we should use javascript to handle this.

@nithinsanjey  FYI

Reply:
Sorry for late reply. I am currently working on implementing some Javascript that'll apply the drop caps to the first letter of a sentence, or something along those lines.

Is that what you were envisioning?

Reply:
yeah. Lets hold this PR until this is done.